PowerEditor/src/functionList.xml

```diff
@@ -936,6 +937,34 @@
 					</functionName>
 				</function>
 			</parser>
+			
+			<!-- ======================================================== [ VHDL ] -->
+			<!--
+			|   Derived from :
+			|       https://community.notepad-plus-plus.org/topic/11554/function-list-for-vhdl
+			\-->
+			<parser
+				displayName="VHDL"
+				id         ="vhdl_syntax"
+			>
+				<function
+					mainExpr="(?x)                                          # free-spacing (see `RegEx - Pattern Modifiers`)
+						  ^                                             # match at beginning of line
+						  \h*                                           # optional leading whitespace
+						  (\w+\h*:)?                                    # ID of process
+						  (\h(?!END)\w+\h)*                             # make sure this is not the end of a module
+						  \h*
+						  (?(1)(ENTITY|BLOCK|PROCESS)|                  # keywords if ':' was detected before

```

Comment:
Likewise, `(?:ENTITY|BLOCK|PROCESS)` to make it non-capturing group.

Reply:
Done, but I can´t see the benefit of non-capturing mode?!?